### DISCUSSION TITLE
Will Vue be supported in the future?

### DISCUSSION THREAD

**Discussion by ghost** [nrwl/nx#9836] [General]
Nx is amazing, but I wonder if vuejs will be supported in the future? I have not seen that in the roadmap. Most of my work is based on Vue, and I would love to see that supported.

**Comment by JFGHT** [comment]
For Googlers; this the only nearly up to date solution I've found: https://github.com/samatechtw/nx-vue3-vite/tree/main

**Comment by louiss0** [comment]
They decided to support Tailwind but Not unoCSS please put a thumbs up for my discussion #19888.
